bundles/org.openhab.binding.wlanthermo/README.md

```diff
@@ -0,0 +1,534 @@
+# WlanThermo Binding
+
+This binding add support for the WlanThermo BBQ Thermometer.  
+| Nano | Mini |
+|---|---|
+| ![WlanThermo Nano V1+](doc/nano.jpg) | ![WlanThermo Mini V2](doc/mini.jpg)
+
+## Supported Things
+
+This binding supports the following version of WlanThermo:
+
+ - Nano V1 (untested, but should be working)
+ - Nano V1+
+ - Mini V1 (untested, but should be working)
+ - Mini V2
+
+ Pitmaster support is currently only enabled for Mini V1/V2.  
+ If you have a WlanThermo Nano with Pitmaster, please contact the author of this binding to include the support in future releases.
+
+## Discovery
+
+There is no auto-discovery of WlanThermo Things. 
+Things must be created manually, e.g. via PaperUI.
+
+## Thing Configuration
+
+WlanThermo things require you to specify the IP-address of your WlanThermo device (the one you enter into your browser to access the WebUI)
+The configuration of username/password is optional. If ommitted, the binding data will be read-only for all channels. WlanThermo Mini things do not require a username/password and will always be read-only!
```

Comment:
Please put each sentence on a single line. That makes reviewing changes more easy.